angular/src/schematics/collection.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "../../node_modules/@angular-devkit/schematics/collection-schema.json",
+  "schematics": {
+  "ng-add": {
```

Comment:
Missed path to the `schema.json`:

"schema": "./add/schema.json",

Reply:
Thanks @vitaliy-bobrov 😄

Reply:
Glad to help! Just watched ngHouston with you ;)